cli/cmd/connect.go

```diff
@@ -21,6 +21,7 @@ import (
 const (
 	usernameEnv = "TT_CLI_USERNAME"
 	passwordEnv = "TT_CLI_PASSWORD"
+	userpassRe  = `[\w-$!%*#?&]+:[\w-$!%*#?&]+`
```

Comment:
Does this charset based on some core tarantool regexps?

Reply:
So what was the inspiration for this particular charset?

Reply:
As an alternative way we could use:

`[^@:/]+:[^@:/]+`

(allow all characters, exclude a special one that configure a path)

Reply:
As far as I understand, the only restriction in `net.box` is separation symbols, and I don't remember any restrictions in user name and password since they are stored as msgpack strings in tuples. So maybe this approach is the best one.

Reply:
Yes, it is look like it:
```
> box.schema.user.create('фывфыв1231*??:(?(*?(')
---
...
```

Reply:
Updated!